doc/src/site/sphinx/deployment/terraform_emr.rst

```diff
@@ -49,10 +51,10 @@ Sparkling Water provides 3 templates/modules:
    - ``aws_secret_key`` (mandatory) - secret key to access AWS
    - ``aws_ssh_public_key`` (optional) - public key (to be able to access EC2 instances via ssh later)
    - ``aws_region`` (optional) - AWS region. Defaults to ``us-east-1``.
-   - ``aws_emr_version`` (optional) - EMR version. Defaults to ``emr-2.20.0``.
+   - ``aws_emr_version`` (optional) - EMR version. Defaults to ``emr-2.26.0``.
```

Comment:
We could do do `SUBST_EMR_VERSION` instead of `emr-2.26.0` so it is less error-prone

Reply:
My original intention was to write details for Spark 2.4 to all SW versions, but you are right these things are easy to forget to change when emr gets upgraded.